drush_language.drush.inc

```diff
@@ -255,9 +268,20 @@ function drush_drush_language_language_import() {
   // Get arguments and options.
   $langcode = array_shift($args);
   $file_paths = $args;
-  // Overwrite existing customized and non-customized translations.
-  $replace_arg = drush_get_option('replace');
-  $overwrite_options = isset($replace_arg) ? ['customized' => TRUE] : ['not_customized' => TRUE];
+
+  // Evaluate overwriting options.
+  $replace_customized = (NULL !== drush_get_option('replace-customized'));
+  $replace_non_customized = (NULL !== drush_get_option('replace-non-customized'));
+  if (NULL !== drush_get_option('replace-customized')) {
```

Comment:
Can't you provide a default value when using ```drush_get_option```? This could simplify the new code a bit. Maybe i'm just entirely wrong